defender-endpoint/linux-preferences.md

```diff
@@ -627,6 +628,7 @@ Determines whether security intelligence updates are installed automatically:
 |**Data type**|Boolean|Drop down|
 |**Possible values**|`true` (default) <p>`false`|Not configured<br>Disabled<br>Enabled (Default)|
 
+Depending on the enforcement level, the automatic security intelligence updates are installed differently. In RTP mode, updates are installed periodically. In Passive/ On-Demand mode updates are installed before every scan.
```

Comment:
Instead of "updates are installed" ==> "Security Intelligence updates are updated/installed before every scan"